client/my-sites/types/controller.jsx

```diff
@@ -38,9 +35,14 @@ export function list( context ) {
 		search: context.query.s
 	};
 
+	// [TODO]: Translate title text when settled upon
 	ReactDom.render(
 		<ReduxProvider store={ context.store }>
-			<Types query={ query } />
+			<div>
```

Comment:
Not sure if the `<div />` is the correct way of doing it within Calypso. Advice needed.